reinforcement_learning/rl_game_server_autopilot/sagemaker/rl_gamerserver_ray.ipynb

```diff
@@ -27,7 +27,7 @@
   },
```

Comment:
<p>supposed to change the param in this cell? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1738/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Updated</p>